### PR TITLE
[dev] - Fixed value to be top aligned vertically in the input field box when used as multiline.

### PR DESCRIPTION
**Pull request by ardneved**

### CODE REVIEW

File: src/modules/text-input/text-input.tsx [hunk 1]
```diff
@@ -58,6 +58,8 @@ const StyledInput = Styled.TextInput<IStyledInput>`
   border-top-width: ${({$isUnderlined}) => ($isUnderlined ? 0 : '1px')};
   border-right-width: ${({$isUnderlined}) => ($isUnderlined ? 0 : '1px')};
   border-left-width: ${({$isUnderlined}) => ($isUnderlined ? 0 : '1px')};
+
```
Comment (raoufr) [inline src/modules/text-input/text-input.tsx:61]: can we remove the empty line?

**Review by raoufr — CHANGES_REQUESTED** [review]
looking good, we are just missing the change log entry for this update. Please let me know if you need guidance to add it. Thanks

**Comment by ajaybushel** [comment]
Do we really need to make changes in the styles? I think this should work even if we just pass the `multiline` prop to the component. 🤔

File: src/modules/text-input/text-input.tsx (hunk 1 above)
Comment (ardneved) [reply src/modules/text-input/text-input.tsx]: sure, it's done

**Comment by ardneved** [comment]
> Do we really need to make changes in the styles? I think this should work even if we just pass the `multiline` prop to the component. 🤔

Yes we do.
please see the [reference](https://reactnative.dev/docs/0.67/textinput#multiline)

**Comment by ardneved** [comment]
> looking good, we are just missing the change log entry for this update. Please let me know if you need guidance to add it. Thanks

@raoufr thank you, yes some guidance for change log entry would be really helpful

**Review by raoufr — APPROVED** [review]
Please add a changelog entry for this fix and we should good to merge

**Comment by ardneved** [comment]
> Please add a changelog entry for this fix and we should good to merge

@raoufr thank you,
change log updated

**Review by KacosPro — APPROVED** [review]
LGTM

**Comment by raoufr** [comment]
Thank you @ardneved  for the updates. We are going to go ahead and merge this one.

**Pull request merged**